### ISSUE TITLE
opt-out / filter CPUCreditBalanceLow alarm

### ISSUE THREAD

**Issue by slang25** [justeat/AwsWatchman#48]
For non t2 EC2 instances, this alarm is always in the `insufficient` state

**Comment by tomhaigh** [comment]
Would it be better to make this alarm opt-in? as using t2 isn't a default choice. Is it too late to change it now?

**Comment by tomhaigh** [comment]
Currently you can set threshold like this:
```
{
    "AlarmName": 400
}
```

This is shorthand for:
```
{
    "AlarmName": { 
         "Threshold": 400,
         // other options here like percentile, number of periods    
    }
}
```

**proposal**

Suggest we allow opt-out by adding another property:

```
{
    "AlarmName": { 
         "Enabled": false,
        // other options here like threshold, percentile, number of periods . not relevant here though
    }
}
```
(Some alarm types in future could have the `Enabled` property defaulted to false so that they become opt-in rather than opt-out.)

We can then make it easier to opt out by introducing another shorthand which would expand to the above:

```
{
    "AlarmName": false
}
```
Thoughts @AnthonySteele @slang25 @andrewclowes ?

**Comment by tomhaigh** [comment]
Didn't get any feedback so have proceeded with PR implementing the above

**Comment by tomhaigh** [comment]
opt out was implemented in #128
